scripts/configure-firefox.in

```diff
@@ -0,0 +1,335 @@
+#!/bin/bash
+
+WorkingDir="@CMAKE_INSTALL_PREFIX@/@TARGET_TOOL_EXEC_FOLDER@/"
+TriggerPort=65432
+MountPoint="user/prefs"
+PrefsSetupMountPoint="system/firefox/prefs"
+ConfigFile="myPrefs.js"
+Decoy="/tmp/imnotreal.js"
+AutoConfigScript="elektra.cfg"
+AutoConfigLauncher="autoconfig.js"
+FFInterceptKey="system/firefox/intercept/running"
+FFInterceptConfig="system/firefox/intercept/userprefs"
+FFInterceptLibDir="system/firefox/intercept/libdir"
+PrefsFile=
+FFLibDir=
+
+testFFRunning()
+{
+    pgrep firefox &>/dev/null
+    echo $?
+}
+
+initialize()
+{
+    kdb check mozprefs &>/dev/null
+
+    hasPrefsPlugin=$?
+
+    if [ "$hasPrefsPlugin" -ne 0 ]; 
+    then
+	echo "Error, mozprefs plugin not found"
+	exit 1
+    fi
+
+    interceptFile="@CMAKE_INSTALL_PREFIX@/lib@LIB_SUFFIX@/libelektraintercept.so"
+
+    if [ ! -e "$interceptFile" ];
+    then
+	echo "Error, libelektraintercept.so not found"
+	exit 1
+    fi
+
+    FFLibDir=$(find /usr/lib /usr/lib32 /usr/lib64 /usr/local/lib -name "firefox*" -type f | xargs dirname 2>/dev/null|sort|uniq)
+    if [ -z "$FFLibDir" ];
+    then
+	echo "Error, couldn't find any Firefox library directory"
+	exit 1
+    fi
+    resNo=$(echo "$FFLibDir" | wc -l)
+    if [ "$resNo" -gt 1 ];
+    then
+	i=1;
+	for line in ${FFLibDir};
+	do
+	    echo "$i) $line"
+	    i=$((++i))
+	done
+
+	echo -n "Select your Firefox library directory: "
+	read -r sel
+
+	FFLibDir=$(sed "${sel}q;d" <<< "$FFLibDir")
+    fi
+
+    echo "Firefox library directory: $FFLibDir"
+    echo
+
+    userConfigDir="${HOME}/.mozilla/firefox/"
+    Profile=$(find "$userConfigDir" -maxdepth 1 -mindepth 1 -type d 2>/dev/null|sort|uniq)
+    if [ -z "$Profile" ];
+    then
+	echo "Couldn't find Firefox profile directory in $userConfigDir"
+	echo -n "Please enter your Firefox profile directory: "
+	read -r Profile
+    fi
+    if [ -z "$Profile" ];
```

Comment:
Did you fix the problem with the user/log keys in the output? Where was the problem?